tools/python/calc_func_checksums.py

```diff
@@ -12,7 +12,7 @@
 
 FUNCTIONS_TO_CALC = [
     # function         checksum variable        func size variable
-    ('func_80019808', 'gFunc80019808Checksum', 'gFunc80019808Length'),
+    ('race_check_finish', 'gFunc80019808Checksum', 'gFunc80019808Length'),

```

Comment:
Should the checkum variable be named `gRaceCheckFinishChecksum` and the func size variable `gRaceCheckFinishFuncLength`?

```suggestion
    ('race_check_finish', 'gRaceCheckFinishChecksum', 'gRaceCheckFinishFuncLength'),
```